gigasecond.md

```diff
@@ -1 +1 @@
-A gigasecond is one billion (10**9) seconds.
+A gigasecond is 10^9 (1,000,000,000) seconds.
```

Comment:
Yeah, I think you are right.

Cohen Carlisle <notifications@github.com> schrieb am Fr., 19. Aug. 2016 um
02:43 Uhr:

> In gigasecond.md
> <https://github.com/exercism/x-common/pull/343#discussion_r75413205>:
>
> > @@ -1 +1 @@
> > -A gigasecond is one billion (10**9) seconds.
> > +A gigasecond is 10^9 (1,000,000,000) seconds.
>
> I think it would be a mistake to try to write exponentiation here in a
> programming language-like syntax. 10^9 is the most straightforward "human"
> way to write 10-to-the-power-of-9. The superscript 9 I'm a little more open
> to, but since 1,000,000,000 appears immediately after 10^9, is there
> really any risk of confusion?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/exercism/x-common/pull/343/files/e034e15b5b42591e4e4393083f1590cab6a0e3de#r75413205>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADmR-htA0XE0MRdhyq46d1S2efJlStrks5qhPw1gaJpZM4JnCuu>
> .
>